.github/workflows/runtests.yml

```diff
@@ -201,8 +201,11 @@ jobs:
       - run: python -m pip install --upgrade pip wheel setuptools
 
       # dependencies to install in all Python versions:
+              aesara wurlitzer autowrap lxml pytest z3                \
```

Comment:
Actually it's more likely this line, but the below blank line shouldn't be there either. The diff of this file should just be to add `z3` and nothing else.